### DISCUSSION TITLE
Actual dependencies, Cuda, Optix

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Bforartists#3298] [General]
Hey,

Just noticed that i might not have the newest dependencies anymore. And i still compile with VS 2019 since the last time i checked VS 2022 required minimum Cuda 11.6, while official Blender still uses Cuda 11.3. Also my Optix lib might be a bit outdated.

@iyadahmed , you have once linked me to the scripts of the blender buildbot, which lists the currently used dependencies. But i cannot find it anymore in my thousands of bookmarks. Can you remember where it was? I would love to use the exact versions than Blender does to have the hightest compatibility.

**Comment by ghost** [comment]
I honestly lost it as well

**Comment by ghost** [comment]
https://wiki.blender.org/wiki/Infrastructure/BuildBot/Changelog

**Comment by ReinerBforartists** [comment]
So it is cuda 11.4 and Optix still 7.3. Many thanks :)
